client/src/partials/hollydays/hollydays.html

```diff
@@ -117,8 +121,11 @@
           <div class="form-group">
             <label class="control-label col-xs-3 required">{{ "COLUMNS.EMPLOYEE_NAME" | translate }}</label>
             <div class="col-xs-8">
-              <select class="form-bhima" required ng-model="session.edit.employee_id" ng-options="employee.id as employee.name + '  ' + employee.postnom + ' ' + employee.prenom for employee in employees.data">
-                <option></option>
+              <select class="form-bhima"
+                ng-model="HolidayCtrl.session.edit.employee_id"
+                ng-options="employee.id as employee.name + '  ' + employee.postnom + ' ' + employee.prenom for employee in HolidayCtrl.employees.data"
+                disabled required>
```

Comment:
For Holiday, we specify holiday data for a specific employee, if we already define an holiday for an employee we can only update label, percent day, date from or date to but not the employee id; if we have to do that, we can simply delete the holiday row and create a new holiday data for another employee...

Require because we need that employee_id